### PR TITLE
Feature/pom and gal 686 716

### PR DESCRIPTION
**Pull request by AlexandraDumitriu**

### CODE REVIEW

File: chemspot.iml [hunk 1]
```diff
@@ -9,76 +9,23 @@
       </configuration>
     </facet>
   </component>
-  <component name="NewModuleRootManager" inherit-compiler-output="false">
+  <component name="NewModuleRootManager" LANGUAGE_LEVEL="JDK_1_5">
```
Comment (ADBalici) [inline chemspot.iml:12]: 1.5? i thought we could do this with 1.8

Comment (ADBalici) [reply chemspot.iml:12]: ah shit this is the .iml. please add the .iml and the .idea directory to gitignore

**Review by ADBalici — CHANGES_REQUESTED** [review]
A couple of things to add to the README:

- can we add the external .jars to GCP and have this pull the jars from there? it would be much easier then
- write down the bash commands or any other steps necessary to build the final .jar (I'll try them out on my machine to see if it works). this will be the foundation of a TeamCity pipeline

File: chemspot.iml (hunk 1 above)
Comment (AlexandraDumitriu) [reply chemspot.iml:12]: We could add these jars as packages in Artifact Registry (see Adding Packages from https://cloud.google.com/artifact-registry/docs/java/manage-packages#console). Then we could import them the way we do galactic4j. If this doesn't work, we can upload them to gcs and download them when needed.
As for bash commands, simply running "mvn prepare-package" should create the jar with dependencies.
To run "mvn package" (which also creates a release zip file with all the resources) make sure you have ids.zip and dict.zip locally and set project.package.data in the pom.xml to their location. I will add this to the README.

**Review by ADBalici — APPROVED** [review]

**Pull request merged**